src/sql/workbench/contrib/notebook/browser/models/cell.ts

```diff
@@ -285,7 +285,8 @@ export class CellModel implements ICellModel {
 
 	private notifyExecutionComplete(): void {
 		if (this._notebookService) {
-			this._notebookService.serializeNotebookStateChange(this.notebookModel.notebookUri, NotebookChangeType.CellExecuted, this);
+			this._notebookService.serializeNotebookStateChange(this.notebookModel.notebookUri, NotebookChangeType.CellExecuted, this)
+				.catch(() => { });
```

Comment:
Looking into a better way of handling errors here. Same for the other catch call.